### PR TITLE
Add command to print out reserve config

### PR DESCRIPTION
**Pull request by DaSichuan**
This doens't work yet. For some reason the clock thing isnt' working, and
I haven't added the part to get the asset symbol.

### CODE REVIEW

**Comment by DaSichuan** [comment]
(WM_print_config) $ target/debug/spl-token-lending show-reserve --reserve           FNNkz4RCQezSSS71rW2tvqZH1LCkTzaiG7Nd1LeA5x5y | python3 -m json.tool
{
    "address": "FNNkz4RCQezSSS71rW2tvqZH1LCkTzaiG7Nd1LeA5x5y",
    "asset": "",
    "collateralMintAddress": "E2PSSXsXJGdpqhhaV3rYPpuy1inRCQAWxcdykA1DTmYr",
    "collateralSupplyAddress": "FiUyeMAnZYkLCbrPGwjJWQwzYJ5AjD6p9N9fx6VxDPMt",
    "liquidityAddress": "HixjFJoeD2ggqKgFHQxrcJFjVvE5nXKuUPYNijFg7Kc5",
    "liquidityFeeReceiverAddress": "DiM1Lf3HiAVpmG3bFNVGXfGaxdUck8oZcTr3tTQfLL6P",
    "userSupplyCap": "10000.000000000000000000"
}

**Comment by nope-finance** [comment]
sorry is this still wip or naw?

File: token-lending/program/src/processor.rs [hunk 1]
```diff
@@ -2116,7 +2116,8 @@ fn get_pyth_product_quote_currency(pyth_product: &pyth::Product) -> Result<[u8;
     Err(LendingError::InvalidOracleConfig.into())
 }
 
-fn get_price(
+/// Gets the price of a token using pyth or switchboard
+pub fn get_price(
```
Comment (nope-finance) [inline token-lending/program/src/processor.rs:2120]: is this needed?

File: token/solana.tokenlist.json [hunk 2]
```diff
@@ -0,0 +1,11650 @@
+{
```
Comment (nope-finance) [inline token/solana.tokenlist.json:1]: maybe can do this as a curl so we dont have to add a large json file we have to update in this repo

**Pull request closed without merging**